web/pgadmin/static/js/components/PgReactDataGrid.jsx

```diff
@@ -119,7 +119,7 @@ export function CustomRow({inTest=false, ...props}) {
 
   return (
     <Row {...props} onKeyDown={handleKeyDown} onCellClick={onCellClick} onCellDoubleClick={onCellDoubleClick}
-      selectCell={(row, column)=>props.selectCell(row, column)} aria-selected={isRowSelected}/>
+      selectCell={(row, column)=>props.selectCell(row, column)} aria-selected={isRowSelected} rangeSelectionMode={props.enableRangeSelection}/>
```

Comment:
this is not needed.

Reply:
thanks, done - I can see that `rangeSelectionMode` is mostly used as to determine the cell range selection in the underlying library, but I'm a little puzzled as to how the cell range selection is working atm without this being toggled active

Reply:
that is taken care at the DataGrid main component level. subcomponents don't need the prop to be passed explicitly.